### ISSUE TITLE
assertHasElementWithAttributes fails if the attribute value contains commas or spaces

### ISSUE THREAD

**Issue by dweingart** [stevegrunwell/phpunit-markup-assertions#13]
**Markup:**

```html
<div>
  <label for="company">Company</label>
  <input required type="text" id="company" name="company" value="Russel, Nader and Kutch">
</div>  
```

**PHP:**

```php
$this->aassertHasElementWithAttributes(['value' => 'Russel, Nader and Kutch'], $html);
```

Expected result is that the test passes. Actual result is that the test fails.

This is due to a bug with the underlying Zend_Dom library: https://github.com/zendframework/zend-dom/issues/17.

**Comment by stevegrunwell** [comment]
@dweingart Thanks for bringing this to my attention and linking to the relevant issue on Zend's repo!

**Comment by stevegrunwell** [comment]
zendframework/zend-dom#20 has been opened in an attempt to address this upstream.

**Comment by stevegrunwell** [comment]
[My pull request was just merged upstream](https://github.com/zendframework/zend-dom/commit/bd79f0e988fb2e5eda39344f9196ea5fb14d9181), so this should be resolved once zendframework/zend-dom 2.7.0 ships.
